### PR TITLE
Audit/trust m12

### PR DESCRIPTION
**Pull request by jankjr**
- [x] implemented suggeted fix
- [x] implemented a test that makes sure withdrawals are larger than deposits if time passes, disabled for stETH as it does not seem to appreciate

### CODE REVIEW

File: test/plugins/individual-collateral/morpho-aave/MorphoAaveV2TokenisedDeposit.test.ts [hunk 1]
```diff
@@ -29,12 +31,12 @@ const mkTestCase = <T extends ITokenSymbol>(
 })
 
 const TOKENS_TO_TEST = [
-  mkTestCase('USDC', '1000.0', '1'),
-  mkTestCase('USDT', '1000.0', '1'),
-  mkTestCase('DAI', '1000.0', '1'),
-  mkTestCase('WETH', '200.0', '1'),
+  // mkTestCase('USDC', '1000.0', '1'),
```
Comment (tbrent) [inline test/plugins/individual-collateral/morpho-aave/MorphoAaveV2TokenisedDeposit.test.ts:34]: I don't think you meant to comment these out?

File: test/plugins/individual-collateral/morpho-aave/MorphoAaveV2TokenisedDeposit.test.ts [hunk 2]
```diff
@@ -233,6 +235,28 @@ const execTestForToken = ({
       await closeTo(methods.balanceUnderlying(alice), fraction(100))
     })
 
+    it('Withdrawals larger than deposits', async () => {
```
Comment (tbrent) [inline test/plugins/individual-collateral/morpho-aave/MorphoAaveV2TokenisedDeposit.test.ts:238]: I just ran this test with the `updateIndexes()` calls commented out, and it passed.

**Review by tbrent — CHANGES_REQUESTED** [review]

File: test/plugins/individual-collateral/morpho-aave/MorphoAaveV2TokenisedDeposit.test.ts (hunk 2 above)
Comment (jankjr) [reply test/plugins/individual-collateral/morpho-aave/MorphoAaveV2TokenisedDeposit.test.ts:238]: Hm, so the change makes no difference?

**Comment by tbrent** [comment]
Turns out this is unneeded, Trust has confirmed

**Pull request closed without merging**